docs/usage-gorilla-mux.md

```diff
@@ -0,0 +1,32 @@
+## Additional steps for users of Gorilla Mux router
+
+If you are using [Gorilla Mux router](https://github.com/gorilla/mux) you must register pprof and gom http handlers.
+
+``` go
+import (
+    "net/http/pprof"
+	gomhttp "github.com/rakyll/gom/http"
+)
+
+func main() {
+    r := mux.NewRouter()
+	attachProfiler(r)
+
+    // ...
+}
+
+func attachProfiler(router *mux.Router) {
+	router.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
I was wondering whether this could be a helper function exported from gom/http/mux.